### PR TITLE
BSWH - connect app add/update references

### PR DESCRIPTION
**Pull request by Gbarra9**
This pull request is related to:

- https://github.com/episphere/connect/issues/885

Code Changes
- Added message for participants with a verification status of outreach timed out.
- Added content HIPAA and Consent forms for BSWH, also positioned the signature, date, and name
- Added Baylor Scott & White Health to select Healthcare Provider dropdown

**Note: Having this pull request reviewable and ready after May release**

### CODE REVIEW

File: js/pages/myToDoList.js [hunk 1]
```diff
@@ -147,36 +147,39 @@ export const myToDoList = async (data, fromUserProfile, collections) => {
                     hideAnimation();
                     return;
                 }
-                else if(data['821247024'] && data['821247024'] == 160161595) {
-                    let site =data['827220437']
+                else if (data[fieldMapping.verification] && data[fieldMapping.verification] == fieldMapping.outreachTimedOut) {
```
Comment (anthonypetersen) [inline js/pages/myToDoList.js:150]: awesome

File: js/pages/samples.js [hunk 2]
```diff
@@ -3,12 +3,12 @@ import { getMyData, hasUserData } from "../shared.js";
 export const renderSamplesPage = async () => {
     document.title = 'My Connect - Samples';
     getMyData().then(res => {
+        console.log("🚀 ~ getMyData ~ res:", res)
```
Comment (anthonypetersen) [inline js/pages/samples.js:6]: needed?

**Review by amber-emmes — APPROVED** [review]
Looks good.

File: js/pages/samples.js [hunk 3]
```diff
@@ -3,12 +3,12 @@ import { getMyData, hasUserData } from "../shared.js";
...
 
         if(!hasUserData(res)) return;
         
-        const site = locations.filter(location => location.concept == res.data['827220437'])[0];
-        
-        let template; 
+        let site = locations.filter(location => location.concept == res.data['827220437'])[0];
```
Comment (anthonypetersen) [inline js/pages/samples.js:10]: could probably add the field id mapping you used in `myToDoList.js`

**Review by anthonypetersen — CHANGES_REQUESTED** [review]
couple of comments added

**Review by anthonypetersen — APPROVED** [review]

**Pull request merged**